### ISSUE TITLE
Option to turn off inline styles to not violate a basic Content Security Policy

### ISSUE THREAD

**Issue by alexkirmse** [mefechoel/svelte-navigator#83]
When using svelte navigator there are a11y elements that are hidden via an inline style. Specifically the [status div](https://github.com/mefechoel/svelte-navigator/blob/main/src/Router.svelte#L196-L203) but others as well.

When using a [basic Content Security Policy that doesn't allow inline styles](https://cheatsheetseries.owasp.org/cheatsheets/Content_Security_Policy_Cheat_Sheet.html#basic-csp-policy), the style won't be applied (showing the status div) and an error message will be logged in the console.

![Screen Shot 2022-07-12 at 2 55 20 PM](https://user-images.githubusercontent.com/12135/178602261-d1e4b522-3fbd-4106-9d06-e0dab54afd82.png)

It would be nice to have a configuration option (like noInlineStyle inside [`a11yConfig`](https://github.com/mefechoel/svelte-navigator/blob/main/src/Router.svelte#L37)) to allow the inline styles to be omitted or to move those styles to `<style></style>` tags since this is in a Svelte component.

**Comment by mefechoel** [comment]
Yes, good catch. That should indeed be configurable. I'll look into it and see what kind of configuration makes the most sense.

**Comment by mefechoel** [comment]
Fixed in 3.2.1. Pass the `disableInlineStyles` prop to your top level Router:

```html
<Router disableInlineStyles>
	<!-- ... -->
</Router>
```

Also make sure to include the `svelte-navigator.css` file somewhere. Either in your apps entrypoint, and let your bundler do the rest:

```js
// In your applications entrypoint, such as `index.js` or `main.js`
import "svelte-navigator/svelte-navigator.css";
```

Or, include the contents of [`svelte-navigator.css`](https://github.com/mefechoel/svelte-navigator/blob/main/svelte-navigator.css) manually in your apps css.
